src/components/VideoTile/VideoTile.tsx

```diff
@@ -192,6 +199,25 @@ const Tile = ({
     ? selectScreenShareByPeerID
     : selectCameraStreamByPeerID;
 
+  function isJSONString(str: string) {
+    try {
+      JSON.parse(str);
+    } catch (e) {
+      return false;
+    }
+    return true;
+  }
+
+  const customerDescription = useHMSStore(
+    selectPeerByID(peer.id),
+  )?.customerDescription;
+
+  const data =
+    customerDescription && isJSONString(customerDescription)
+      ? JSON.parse(customerDescription)
+      : undefined;
+  const isHandRaised = typeof data === 'object' ? data.raiseHand : false;
```

Comment:
move all of the above to one function. 
const isHandRaised = getHandRaised() which returns true/false